src/test/kotlin/app/batch/DecompressionProcessorTest.kt

```diff
@@ -12,37 +11,16 @@ import com.nhaarman.mockitokotlin2.verify
 import org.apache.commons.compress.compressors.CompressorStreamFactory
 import org.apache.commons.compress.compressors.gzip.GzipCompressorInputStream
 import org.apache.commons.io.IOUtils
-import org.apache.hadoop.hbase.client.Connection
 import org.junit.Assert.assertEquals
 import org.junit.Assert.assertTrue
 import org.junit.Test
-import org.junit.runner.RunWith
 import org.slf4j.LoggerFactory
-import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.boot.test.mock.mockito.MockBean
-import org.springframework.test.context.ActiveProfiles
-import org.springframework.test.context.TestPropertySource
-import org.springframework.test.context.junit4.SpringRunner
 import java.io.ByteArrayInputStream
 import java.io.ByteArrayOutputStream
 import java.nio.charset.StandardCharsets
 
-@RunWith(SpringRunner::class)
-@ActiveProfiles("awsS3")
-@SpringBootTest
-@TestPropertySource(properties = [
-    "hbase.zookeeper.quorum=hbase",
-    "aws.region=eu-west-1",
-    "s3.bucket=bucket1",
-    "s3.prefix.folder=test/output/",
-    "s3.key.regex=([A-Za-z]*\\.[A-Za-z]*\\.[0-9]{4}\\.json\\.gz)",
-    "s3.data.key.extension=\\.enc$",
-    "s3.metadata.key.extension=\\.encryption\\.json$"
-])
-class DecompressionProcessorTest {
 
-    @MockBean
-    private lateinit var connection: Connection
+class DecompressionProcessorTest {
```

Comment:
Should_Throw_Exception_When_Given_Non_Gzipped_Stream and other tests - can we assert the exception message please, same as other cases?

Reply:
Added